packages/dashboard-core-plugins/src/GridPlugin.tsx

```diff
@@ -29,14 +30,26 @@ export const GridPlugin = (props: GridPluginProps): JSX.Element => {
     hydrate,
     theme,
   } = props;
-  const handleOpen = useCallback(
-    (
-      title: string,
-      makeModel: () => IrisGridModel,
-      metadata: Record<string, unknown> = {},
+  const handlePanelOpen = useCallback(
+    ({
+      dragEvent,
+      fetch,
       panelId = shortid.generate(),
-      dragEvent?: DragEvent
-    ) => {
+      widget,
+    }: {
+      dragEvent?: DragEvent;
+      fetch: () => Promise<Table>;
+      panelId?: string;
+      widget: VariableDefinition;
+    }) => {
+      const { name, type } = widget;
+      if (type !== dh.VariableType.TABLE) {
```

Comment:
I see there's also `dh.VariableType.TREETABLE` type, should this plugin handle it as well?

Reply:
We don't have tree tables yet but I should be listening for it in anticipation.